packages/@ama-sdk/generator-sdk/package.json

```diff
@@ -78,5 +86,7 @@
     "node": ">=14.0.0",
     "npm": ">=4",
     "yarn": "<4.0.0"
-  }
+  },
+  "main": "./dist/generators/sdk-generator.js",
```

Comment:
I think it's without `./dist` and I am not sure it is needed

Reply:
Yeah it can probably be removed as this is not the way the generator shall be imported